### PR TITLE
add single eq ODESystem constructor

### PR DESCRIPTION
**Pull request by anandijain**
should be harmless additional constructor. there doesn't seem to be any reason not to have this.

### CODE REVIEW

File: src/systems/diffeqs/odesystem.jl [hunk 1]
```diff
@@ -208,3 +208,5 @@ function flatten(sys::ODESystem)
                         )
     end
 end
+
+ODESystem(eq::Equation; kwargs...) = ODESystem([eq]; kwargs...)
```
Comment (YingboMa) [inline src/systems/diffeqs/odesystem.jl:212]: It should be
```suggestion
ODESystem(eq::Equation, args...; kwargs...) = ODESystem([eq], args...; kwargs...)
```

**Comment by YingboMa** [comment]
Also, could you add a test?

**Comment by anandijain** [comment]
thanks @YingboMa ! I don't have a great intuition for what is a good test. happy to revise this

**Comment by YingboMa** [comment]
Add a single equation system like `u' = alpha*u` and test for `equations(sys)`.

**Comment by anandijain** [comment]
hows this

File: test/odesystem.jl [hunk 2]
```diff
@@ -267,7 +267,10 @@ eqs = D(x) .~ nums
 
 systems = ODESystem.(eqs)
 sys = ODESystem(eqs[1])
+eq = D(x) ~ r*x
```
Comment (YingboMa) [inline test/odesystem.jl:270]: This should be moved to the end to make definition and testing local.

File: test/odesystem.jl [hunk 3]
```diff
@@ -256,7 +256,7 @@ sys = ODESystem(eqs, t)
 @test isempty(parameters(sys))
 
 # one equation ODESystem test
-@parameters t
+@parameters t r
```
Comment (YingboMa) [inline test/odesystem.jl:259]: so does this

**Comment by anandijain** [comment]
thanks. i think this might be a better test. if this works then so will the broadcast, and this is more readable

**Pull request merged**